SpotifyInterface.c

```diff
@@ -346,15 +346,21 @@ bool Spotify_SendCommand(SpotifyInterfaceHandler_t SpotifyInterfaceHandler, int
             break;
 
         case GetUserInfo:
+        //SEZ@Done ESP_LOGW just for test.
+        ESP_LOGW(TAG, "GetUserInfo");
             Spotify_GetInfo(Command, PrivateHandler.token.AccessToken);
-            IsSuccessfull = PrivateHandler.SpotifyBuffer.status == 200;
+        ESP_LOGW(TAG, "Spotify_GetInfo");    
+            IsSuccessfull = (PrivateHandler.SpotifyBuffer.status == 200)? true:false;
```

Comment:
what is the reason for this change? previous code also had same result
if the `PrivateHandler.SpotifyBuffer.status` is equal to `200`, retun `true`, if not retun `false`

Reply:
![Screenshot 2024-06-26 104938](https://github.com/Behn4m/SpotifyInterface/assets/6756035/abc08c03-5d4d-4d1c-acf5-6e5a540b3ac4)
🙄

Reply:
![image](https://github.com/Behn4m/SpotifyInterface/assets/6756035/ca10b55f-b26f-429d-b98f-f35a3c11fb08)
🙄😊